rails/config/application.rb

```diff
@@ -144,6 +144,9 @@ class Application < Rails::Application
       end
     end
 
+    # Add a middlewere to log more info about the response
+    config.middleware.insert_before 0, "Rack::ResponseLogger"
+
```

Comment:
Did you check if this effects the output same site cookies gem?  I wouldn't think it would but that is also some Rack middleware and adding this might cause some of the magic dust to escape.

Reply:
I didn't check that.  Do you remember how to check it?